### ISSUE TITLE
Can't autocomplete css-next `vars` and `mixins`

### ISSUE THREAD

**Issue by weichensw** [atom/autocomplete-css#56]
```
.class {
  background-color: var(--custom-color);
  @apply(--custom-mixin);
}
```

The default provider is at least collecting the part after `--`. However css provider disabled default provider.

atom/autocomplete-plus#740

**Comment by WoodyWoodsta** [comment]
Given that https://github.com/atom/language-css/pull/99 was recently merged and is in the pipeline for Atom 1.15, what is the status of this issue? It would be great if custom variable auto-completion could accompany the added support for that syntax in `language-css`.

**Comment by winstliu** [comment]
It may be feasible now, but someone will probably have to create a PR for it.

**Comment by WoodyWoodsta** [comment]
I don't mind having a go, after I understand whether or not https://github.com/atom/autocomplete-css/pull/57 is a complete fix as was indicated.

**Comment by winstliu** [comment]
I don't believe #57 actually fixes this.  All it does is allow the default provider to also provide suggestions, which I think is undesirable since that introduces the possibility of nonsensical suggestions (such as `display: background-image` instead of `display: block`).
